Cargo.lock

```diff
@@ -429,8 +428,11 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "memoffset"
-version = "0.2.1"
+version = "0.5.1"
```

Comment:
This is the main point of the update: old `memoffset` is broken in many ways, and I am trying to remove it from the rustc dependency chain.